.github/workflows/build.yml

```diff
@@ -60,9 +82,13 @@ jobs:
         with:
           node-version: ${{ env.NODE_VERSION }}
       - name: Install Firebase CLI
-        run: npm install -g firebase-tools@12.7.0
-      - name: Deploy to Firebase hosting
-        run: make deploy
+        run: npm install -g firebase-tools@12.9.1
+      - uses: dart-lang/setup-dart@b64355ae6ca0b5d484f0106a033dd1388965d06d
+        with:
+          sdk: stable
+      - run: tool/check-links.sh   
```

Comment:
For my understanding, we want to check the links both when the PR is opened and staging is triggered, and when it is merged?

Reply:
That's what has been done traditionally. It's not super essential but it can help occasionally since we don't use a merge queue currently.